src/main/resources/assets/admin/common/js/ai/AiTool.ts

```diff
@@ -0,0 +1,10 @@
+
+export enum AiTool {
+    OPEN_AI_DIALOG,
+    AI_STATE,
+}
```

Comment:
I'd suggest adding string values to enum entries. It would better for debugging to see string values somewhere down the flow, instead of just `0`.
Otherwise, it will be compiled into:
```js
(function (AiTool) {
    AiTool[AiTool["OPEN_AI_DIALOG"] = 0] = "OPEN_AI_DIALOG";
    AiTool[AiTool["AI_STATE"] = 1] = "AI_STATE";
})(AiTool || (AiTool = {}));
```
instead of 
```js
(function (AiTool) {
    AiTool["OPEN_AI_DIALOG"] = "openAiDialog";
    AiTool["AI_STATE"] = "aiState";
})(AiTool || (AiTool = {}));
```